src/js/pages/Topics.js

```diff
@@ -6,7 +6,7 @@ import TileGroup from 'js/modules/TileGroup';
 
 const Topics = ({ allTopics }) => {
   const links = allTopics.edges.map(topic => ({
-    url: `/topics/${topic.node.id}`,
+    url: `/topics/${topic.node.slug}`,
```

Comment:
same as above, we have a very similar method that does this mapping in cleanData.js